examples/com.sap.vocabularies.UI.v1.DataFieldDefault-sample.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/UI.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.UI.v1" Alias="UI" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Common.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Common.v1" Alias="Common" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema Namespace="datafielddefault.sample" xmlns="http://docs.oasis-open.org/odata/ns/edm">
+      <Annotations Target="my.container/QueryResult/PostingDate">
+        <!-- Based on these annotations, the filter facet for the PostingDate property shall, in addition to the usual date picker,
+             offer two input helps
+             * one computes the PostingDate based on a fiscal date function that is chosen from the FiscalDateFunctions entity set, e.g., "current period"
+             * one computes the PostingDate based on a Gregorian date function that is chosen from the GregorianDateFunctions entity set, e.g., "today".
+             In both cases the computation happens on the server, when an OData request like in com.sap.vocabularies.UI.v1.SelectionVariant-sample.xml is executed.
+        -->
+        <Annotation Term="UI.DataFieldDefault" Qualifier="FiscalDateFunction">
```

Comment:
why do you use DataFieldDefault here? Does the user have the choice which one he wants to see in the UI, or do you want offer all in one?

Reply:
Both fiscal and Gregorian date functions shall be offered to the user (in addition to the usual date picker).